library/std/src/time.rs

```diff
@@ -544,7 +527,7 @@ impl Add<Duration> for SystemTime {
     /// This function may panic if the resulting point in time cannot be represented by the
     /// underlying data structure. See [`checked_add`] for a version without panic.
     ///
-    /// [`checked_add`]: ../../std/time/struct.SystemTime.html#method.checked_add
+    /// [`checked_add`]: SystemTime::checked_add
```

Comment:
And here, it would be clearer. Reading the code directly is way better now, also the docs will be more explicit.

Reply:
Same as above